reactjs-films-homework/src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,8 +1,10 @@
 /* eslint-disable array-callback-return */
-import React, { useState, useEffect } from 'react';
+import React, { useState, useMemo } from 'react';
 import img from './img/Movie-card.png';
 import styles from './MovieCard.module.scss';
-import mockGenres from '../../mocks/genresIDs';
+import { useAppSelector, useAppDispatch } from '../../hooks/hooks';
+import { setMovieID } from '../../store/rootStore/movieDetailsPageStore/movieDetailsPageSlice';
+import GenresList from './GenresList';
 
 export interface ICard {
```

Comment:
what do you need this interface? You can keep just 'IMovieCard'

Reply:
This interface is imported to 'movieDetailsPageSlice.ts', and  'appFetchMovieGenre' should be typed of ICard. Maybe I should move it to the 'movieDetailsPageSlice.ts'?

Reply:
It's not good to import data-related types to the slice from the component file. High-level modules shouldn't depend on low level modules. So that it's better to move all such types to src/types or src/store/types